Raven.Studio.Html5/App/commands/database/documents/getCollectionLabelCommand.ts

```diff
@@ -0,0 +1,35 @@
+﻿import commandBase = require("commands/commandBase");
+import database = require("models/resources/database");
+
+class getCollectionLabelCommand extends commandBase {
```

Comment:
Please review:
studioConfig.ts, databaseStudioConfig.ts + shell.ts::fetchStudoConfig 

That should help.